package.json

```diff
@@ -15,7 +15,8 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "grunt": "0.4.5",
+    "babel-core": "^6.5.2",
```

Comment:
Why do we need Babel? Are you using any ES2015 somewhere? Sorry, asking without checking the other code commits.